dali/operators/image/paste/multipaste.cc

```diff
@@ -26,23 +26,30 @@ This operator can also change the type of data.)code")
 .InputDox(0, "images", "3D TensorList", R"code(Batch of input images.
 
 Assumes HWC layout.)code")
-.AddArg("in_ids", R"code(1D TensorList of type int.
+.AddArg("in_ids", R"code(Indices of the inputs to paste data from.)code",
+        DALI_INT_VEC, true)
+.AddOptionalArg<int>("in_anchors", R"code(Absolute coordinates of LU corner
+of the source region.
 
-Indexes from what inputs to paste data in each iteration.)code", DALI_INT32, true)
-.AddOptionalArg<int>("in_anchors", R"code(2D TensorList of type int64
+2D data with the first dimension containing as many entries as elements in ``in_ids``
+and the second one containing as many elements as the number of dimensions of the data.
```

Comment:
```suggestion
2D data with where first dimension corresponds to the elements of ``in_ids``
and the second one equal to the number dimensions of the data.
```
Dimension is a number, it doesn't contain anything.